### PR TITLE
add helm chart for global error pages

### PR DESCRIPTION
**Pull request by rpahli**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://developercertificate.org) signed
- [x] Chart Version bumped (if the pr is an update to an existing chart)
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by axdotl** [comment]
README missing

File: charts/error-pages/README.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Global Error Pages
+
+## Introduction
+
+This chart installs a global error controller for an [error page for traefik](https://docs.traefik.io/configuration/backends/kubernetes/#global-default-backend-ingresses).
```
Comment (axdotl) [inline charts/error-pages/README.md]: Maybe re-phrase this, proposal:
This chart installs a controller for [Traefik](https://traefik.io) global [error pages]((https://docs.traefik.io/configuration/backends/kubernetes/#global-default-backend-ingresses).

File: charts/error-pages/README.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Prerequisites
+
+-   Has been tested on Kubernetes 1.11+
+
+## Installing the Chart
+
+To install the chart you have to set `objStoreConfig`.
```
Comment (axdotl) [inline charts/error-pages/README.md]: where and how to to this?

File: charts/error-pages/README.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+## Configuration
+
+The following table lists the configurable parameters of the error-pages chart and their default values.
+
+| Parameter                                  | Description                               | Default                            |
+| ------------------------------------------ | ----------------------------------------- | ---------------------------------- |
+| `replicaCount` | replica count | `1`|
+| `updateStrategy` | Deployment update strategy | `type: RollingUpdate` |
+| `image.repository` | Docker image repo | `improbable/thanos`|
```
Comment (monotek) [inline charts/error-pages/README.md]: docker image is thanos

**Pull request merged**